### PR TITLE
How To Read Logs guide

### PR DESCRIPTION
**Pull request by melekes**
Refs #619

### CODE REVIEW

File: docs/how-to-read-logs.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+"Started node" message signals that everything is ready for work.
+
+::
+
+    I[10-04|13:54:30.391] Starting RPC HTTP server on tcp socket 0.0.0.0:46657 module=rpc-server
+    I[10-04|13:54:30.392] Started node                                 module=main nodeInfo="NodeInfo{pk: PubKeyEd25519{DF22D7C92C91082324A1312F092AA1DA197FA598DBBFB6526E177003C4D6FD66}, moniker: anonymous, network: test-chain-3MNw2N [remote , listen 10.0.2.15:46656], version: 0.11.0-10f361fc ([wire_version=0.6.2 p2p_version=0.5.0 consensus_version=v1/0.2.2 rpc_version=0.7.0/3 tx_index=on rpc_addr=tcp://0.0.0.0:46657])}"
+
+Next follows a standard block creation cycle, where we enter a new round,
+propose a block, receive more than 2/3 of prevotes, than precommits and finally
+have a chance to commit a block. For details, please refer to `Consensus
```
Comment (zramsay) [inline docs/how-to-read-logs.rst]: then precommits?

File: docs/how-to-read-logs.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+Overview
+<https://tendermint.readthedocs.io/projects/tools/en/master/introduction.html#consensus-overview>__`
+or `Byzantine Consensus Algorithm
+<https://tendermint.readthedocs.io/projects/tools/en/master/specification.html>__`.
```
Comment (zramsay) [inline docs/how-to-read-logs.rst]: relative links

**Comment by melekes** [comment]
Is `/introduction.html#consensus-overview` a correct form? Is there a way for me to check the result before it will be merged into master?

**Comment by zramsay** [comment]
`cd docs && make html && open _build/html/index.html`

**Comment by zramsay** [comment]
we also have a develop branch on RTD

**Pull request merged**